pkg/apis/apis_test.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2018 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apis
+
+import (
+	"testing"
+
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func TestAddToScheme(t *testing.T) {
+	type args struct {
+		s *runtime.Scheme
+	}
```

Comment:
I always find this `args` sub-struct a little redundant, but I know this is how https://github.com/cweill/gotests generates test scaffolding so I could see a case either way for it.